scalability_and_performance/managing-bare-metal-hosts.adoc

```diff
@@ -0,0 +1,26 @@
+[id="managing-bare-metal-hosts"]
+= Managing bare metal hosts
+include::modules/common-attributes.adoc[]
+:context: managing-bare-metal-hosts
+
+toc::[]
+
+When you install {product-title} on a bare metal cluster, you can provision and manage bare metal nodes using `machine` and `machineset` custom resources (CRs) for bare metal hosts that exist in the cluster.
+
+include::modules/about-bare-metal-hosts-and-nodes.adoc[leveloffset=+1]
+include::modules/maintaining-bare-metal-hosts.adoc[leveloffset=+1]
+
+.Additional Resources
+
+* xref:../machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc[Adding compute machines to bare metal]
+
+include::modules/adding-bare-metal-host-to-cluster-using-web-console.adoc[leveloffset=+2]
+include::modules/adding-bare-metal-host-to-cluster-using-yaml.adoc[leveloffset=+2]
+include::modules/automatically-scaling-machines-to-available-bare-metal-hosts.adoc[leveloffset=+2]
+
+.Additional resources
+
+* xref:../installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc[Expanding the cluster]
```

Comment:
* xref:../installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc#ipi-install-expanding-the-cluster[Expanding the cluster]